clients/src/main/java/org/apache/kafka/common/requests/SyncGroupResponse.java

```diff
@@ -19,14 +19,12 @@
 import org.apache.kafka.common.protocol.ApiKeys;
 import org.apache.kafka.common.protocol.Errors;
 import org.apache.kafka.common.protocol.ProtoUtils;
-import org.apache.kafka.common.protocol.types.Schema;
 import org.apache.kafka.common.protocol.types.Struct;
 
 import java.nio.ByteBuffer;
 
 public class SyncGroupResponse extends AbstractResponse {
```

Comment:
I think we should document it in `Protocol.java` as that is used to generate the protocol documentation on the website. In another PR, please. :)